### PR TITLE
Support dedicated hypervisor

### PR DESCRIPTION
**Pull request by kazyshr**
Support all APIs for dedicated hypervisor.
https://ecl.ntt.com/en/documents/api-references/dedicated-hypervisor/index.html

### CODE REVIEW

**Review by ghost — APPROVED** [review]
LGTM:)

File: ecl/dedicated_hypervisor/v1/license_types/testing/fixtures.go [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+var FirstLicenseType = license_types.LicenseType{
+	ID:            "9c54c437-5f0f-46f5-8270-ddf450a44135",
+	Name:          "Windows Server 2016 Standard Edition",
+	HasLicenseKey: false,
+	Unit:          "VM",
+	LicenseSwitch: true,
+	Description:   "Windows Server 2016 Standard Edition",
+}
+
+// SecondUsage is the second LicenseType in the List request.
```
Comment (keiichi-hikita) [inline ecl/dedicated_hypervisor/v1/license_types/testing/fixtures.go]: looks typo

File: ecl/dedicated_hypervisor/v1/license_types/testing/fixtures.go [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+			"id": "e37c05ba-8fd0-493e-93d2-688833363a74",
+			"license_switch": false,
+			"name": "vCenter Server 6.x Standard",
+			"unit": "License"
+		}
+	]
+}
+`
+
+// FirstUsage is the first LicenseType in the List request.
```
Comment (keiichi-hikita) [inline ecl/dedicated_hypervisor/v1/license_types/testing/fixtures.go]: looks typo

File: ecl/dedicated_hypervisor/v1/license_types/testing/fixtures.go [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+	ID:            "9c54c437-5f0f-46f5-8270-ddf450a44135",
+	Name:          "Windows Server 2016 Standard Edition",
+	HasLicenseKey: false,
+	Unit:          "VM",
+	LicenseSwitch: true,
+	Description:   "Windows Server 2016 Standard Edition",
+}
+
+// SecondUsage is the second LicenseType in the List request.
+var SecondLicenseType = license_types.LicenseType{
```
Comment (keiichi-hikita) [inline ecl/dedicated_hypervisor/v1/license_types/testing/fixtures.go:49]: looks typo

File: ecl/dedicated_hypervisor/v1/license_types/testing/fixtures.go [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+	ID:            "e37c05ba-8fd0-493e-93d2-688833363a74",
+	Name:          "vCenter Server 6.x Standard",
+	HasLicenseKey: true,
+	Unit:          "License",
+	LicenseSwitch: false,
+	Description:   "vCenter Server 6.x Standard",
+}
+
+// ExpectedUsagesSlice is the slice of LicenseTypes expected to be returned from ListResult.
```
Comment (keiichi-hikita) [inline ecl/dedicated_hypervisor/v1/license_types/testing/fixtures.go]: same as the above

File: ecl/dedicated_hypervisor/v1/licenses/results.go [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+// results.
+func ExtractLicenses(r pagination.Page) ([]License, error) {
+	var s struct {
+		Licenses []License `json:"licenses"`
+	}
+	err := (r.(LicensePage)).ExtractInto(&s)
+	return s.Licenses, err
+}
+
+// Extract interprets any commonResult as a License.
```
Comment (keiichi-hikita) [inline ecl/dedicated_hypervisor/v1/licenses/results.go]: First word of comment block does not match to actual function name.

File: ecl/dedicated_hypervisor/v1/licenses/testing/fixtures.go [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+			"key": "0021L-8CJ47-2829A-0A8K2-CXN4J",
+			"assigned_from": "2017-06-01T04:13:31Z",
+			"expires_at": null,
+			"license_type": "vCenter Server 6.x Standard"
+		}
+	]
+}
+`
+
+const GetResult = `
```
Comment (keiichi-hikita) [inline ecl/dedicated_hypervisor/v1/licenses/testing/fixtures.go:38]: no comment block.

**Comment by kazyshr** [comment]
Fix typo and missing comments.

**Review by keiichi-hikita — APPROVED** [review]
LGTM!

**Pull request merged**